npachpande/async/miawCustomPrechat/customPrechatForm.css

```diff
@@ -0,0 +1,13 @@
+:host {
+    display: flex;
```

Comment:
just some basic styling here. planning to just add a note about `configuration.branding` so that customers can consume the branding information from config as they seem fit.